<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P76104833-hw1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>1. Camera Calibration </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btn1_1">
         <property name="text">
          <string>Corner detection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1_2">
         <property name="text">
          <string>intrinsic matrix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1_3">
         <property name="text">
          <string>extrinsic matrix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1_4">
         <property name="text">
          <string>distortion matrix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>undistorted result</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>2. Augmented Reality </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn3_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>2.1 Show words on board</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn3_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>3.2 Video Tracking</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>3. Stereo Disparity Map </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="btn4_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>3.1 Stereo Disparity Map</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
